src/applications/hca/helpers.jsx

```diff
@@ -1425,3 +1426,60 @@ export const medicarePartADescription = (
     </div>
   </div>
 );
+
+/**
+ *
+ * Provides the current Central Time (CT) offset according to whether or not daylight savings is in effect
+ * @export
+ * @param {boolean} isDST
+ * @returns {number} offset in minutes
+ */
+export function getCSTOffset(isDST) {
+  const offsetHours = isDST ? 5 : 6;
+  return offsetHours * 60;
+}
+
+/**
+ *
+ * Converts a timezone offset into milliseconds
+ * @export
+ * @param {number} offset (in minutes)
+ */
+export function getOffsetTime(offset) {
+  return 60000 * offset;
+}
+
+/**
+ *
+ * Determines UTC time from a local time and its offset
+ * @export
+ * @param {number} time (in milliseconds)
+ * @param {number} offset (in minutes)
+ */
+export function getUTCTime(time, offset) {
+  return time + (getOffsetTime(offset));
+}
+
+/**
+ * Provides a current date object in Central Time (CT)
+ * Adapted from https://stackoverflow.com/a/46355483 and https://stackoverflow.com/a/17085556
+ */
+export function getCSTDate() {
+  const today = new Date();
+  const isDST = moment().isDST();
+  const cstOffset = getCSTOffset(isDST);
+
+  const utc = getUTCTime(today.getTime(), today.getTimezoneOffset());
+
+  return new Date(utc + getOffsetTime(cstOffset));
```

Comment:
Why is the timezone offset being compensated for twice: once in the call to `getUTCTime` and again here?

Reply:
There are two offsets, one is for local time, and one is for Central Time (CT).

The local time offset is used to determine UTC time by adding the local time offset to local time.

The CT offset is used to subtract from UTC, in order to determine Central Time.

I've added additional comments and rearranged things to make this clearer.